subtest/scenario.py

```diff
@@ -685,3 +688,44 @@ def shorten_dataset_proportion(self):
 
         self.dataset.x_train = x_train[train_idx[0:skip_idx]]
         self.dataset.y_train = y_train[train_idx[0:skip_idx]]
+
+
+def run_scenario(current_scenario):
+
```

Comment:
I am not too convinced by putting this in the `scenario` file. This is really a kind of core usage of all the library. It would be clearer to have it in a dedicated file I think

Reply:
It would totally fit into a dedicated file (maybe `utils.py` ?)

What do you think ?

Reply:
It could be a method of the Scenario class as well